docs/guides/latest-updates.mdx

```diff
@@ -14,22 +14,24 @@ Keep up with the latest and greatest from Qiskit and IBM Quantum&trade;! Gathere
 
 {/* release summaries written by product owners */}
 
-### Qiskit SDK v1.1.0
+### Qiskit SDK v1.2.0
 
 *To see the release notes for all versions, visit the [Qiskit SDK release notes](/api/qiskit/release-notes).*
 
-Qiskit SDK v1.1.0 has arrived, marking the first minor version release of the Qiskit SDK’s first major version. On the IBM Quantum blog, we review some of the key highlights — including top new features and improvements, as well as some important deprecations. Read the blog [here](https://www.ibm.com/quantum/blog/qiskit-1-1-release-summary).
+Qiskit 1.2 (stable release) is now available on [PyPI](https://pypi.org/project/qiskit/1.2.0/) and [GitHub](https://github.com/Qiskit/qiskit/releases/tag/1.2.0). Thanks to the whole dev team and all external contributors!
 
-The updates and improvements include the following:
+- Many of the core structures have been moved to Rust, allowing for significant speedups mostly in the construction, manipulation (including copy and parameter assignment), and synthesis of circuits.
+- Better compiler quality due to the addition of unitary peephole optimization (for optimization levels 2 and 3) and improvements in Sabre.
+- [`generate_preset_pass_manager`](/api/qiskit/transpiler_preset#generate_preset_pass_manager) importance highlight, which is now importable from the top level.
 
-- Updated `QuantumCircuit` objects that now support typed classical variables
-- Improvements to the preset transpiler pass managers
-- Speedups in transpilation runtime, thanks to our continued work re-writing transpiler passes in Rust
-- A number of new transpiler passes, including `ElidePermutation` and `StarPreRouting`
+Important deprecations:
+- Primitive V1 reference implementations (like `qiskit.primitives.Estimator`) and tooling are deprecated in favor of V2 Primitives. The Primitive V1 base classes remain available.
+- Backend V1 is now deprecated in favor of [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2), including the deprecation of `Qobj`, `assemble`, and other scaffolding related to the old V1.
```

Comment:
```suggestion
- [`BackendV1`](/api/qiskit/qiskit.providers.BackendV1) is now deprecated in favor of [`BackendV2`](/api/qiskit/qiskit.providers.BackendV2), including the deprecation of `Qobj`, `assemble`, and other scaffolding related to the old V1.
```

Reply:
Is it ok to point to something that's deprecated, especially if we're not also linking to the pages for other deprecations `Qobj` and `assemble`?

Reply:
Yeah because this page won't live forever, so it's fine to link to it. I think it can be helpful so that people can see what the old interface was.

> especially if we're not also linking to the pages for other deprecations Qobj and assemble?

fwiw, those could also be linked to